fleets/day2/chart-templates/rancher-turtles/fleet.yaml

```diff
@@ -0,0 +1,69 @@
+defaultNamespace: rancher-turtles-system
+
+helm:
+  releaseName: rancher-turtles
+  chart: "oci://registry.suse.com/edge/3.1/rancher-turtles-chart"
+  version: "0.3.2"
+  # custom chart value overrides
+  values: {}
+
+diff:
```

Comment:
Why do we need all of those? Is there an issue that I can go through to better understand the list below?